#! /bin/sh /usr/share/dpatch/dpatch-run
## 72_hyperarch_error_handling.dpatch by  <lionel@mamane.lu>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Debian bug #310451
## DP: Add error handling to HyperArch

@DPATCH@
diff -urNad mailman-2.1.5~/Mailman/Archiver/HyperArch.py mailman-2.1.5/Mailman/Archiver/HyperArch.py
--- mailman-2.1.5~/Mailman/Archiver/HyperArch.py	2003-12-26 21:41:30.000000000 +0100
+++ mailman-2.1.5/Mailman/Archiver/HyperArch.py	2005-11-13 17:53:49.260054232 +0100
@@ -43,6 +43,7 @@
 
 from Mailman import mm_cfg
 from Mailman import Utils
+from Mailman import Errors
 from Mailman import LockFile
 from Mailman import MailList
 from Mailman import i18n
